src/components/text-input/index.md.njk

```diff
@@ -59,6 +59,19 @@ Fluid width inputs will resize with the viewport.
 
 {{ example({group: "components", item: "text-input", example: "input-fluid-width", html: true, nunjucks: true, open: false, size: "xl"}) }}
 
+### Numbers
+
+If it’s likely that the user will need to enter a number and you want to bring up the numeric keypad on a mobile device, use the following approach:
+
+{{ example({group: "components", item: "text-input", example: "number-input", html: true, nunjucks: true, open: false, size: "m"}) }}
+
+Avoid using `input type=”number”` unless your user research shows that there’s a need for it. With `input type=”number”` there’s a risk of users accidentally incrementing a number when trying to navigate between page elements. And if the user tries to enter something that’s not a number, there’s no explicit feedback about what they’re doing wrong.
```

Comment:
How about ‘Do not use’ instead of ‘Avoid using’ as it's stronger/clearer?

Reply:
Also, I think users will want to know why navigating between page elements can cause this problem to occur. Can you elaborate here?